ci/doc/self-assessment-before-submitting-pr.md

```diff
@@ -22,3 +13,84 @@ $ sudo pbuilder create
 $ pdebuild  # build package to generate *.deb file
 $ ls -al /var/cache/pbuilder/result/*.deb
 ```
+
+# How to build [Tizen](https://source.tizen.org/documentation/reference/git-build-system/usage/gbs-build) RPM package
+You have to execute ***gbs build*** command before submitting your PR.
+```bash
+$ sudo vi /etc/apt/sources.list.d/tizen.list
+deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_16.04/ / # upgraded to xenial
+$ sudo apt update
+$ sudo install gbs
+$ cp TAOS-CI/packaging/.gbs.conf ~/
+$ time gbs build -A x86_64  --clean --include-all  # Generate *.rpm from source for x86_64
+$ time gbs build -A aarch64 --clean --include-all  # Generate *.rpm from source for aarch64
+```
+
+# How to build [Yocto](https://wiki.yoctoproject.org/wiki/Application_Development_with_Extensible_SDK
+) DEB package
```

Comment:
This line looks strange. 
especially, ')' is strange

Reply:
Nice catch. Fixed.